receiver/prometheusreceiver/internal/metricfamily.go

```diff
@@ -319,3 +339,47 @@ func (mf *metricFamily) appendMetric(metrics pmetric.MetricSlice) {
 
 	metric.MoveTo(metrics.AppendEmpty())
 }
+
+func (mf *metricFamily) addExemplar(l labels.Labels, e exemplar.Exemplar) {
+	gk := mf.getGroupKey(l)
+	es, ok := mf.exemplars[gk]
+	if !ok {
+		es = pmetric.NewExemplarSlice()
+	}
+	addExemplarToSlice(e, es.AppendEmpty())
+	if es.Len() > 0 {
+		mf.exemplars[gk] = es
+	}
+}
+
+func addExemplarToSlice(pe exemplar.Exemplar, e pmetric.Exemplar) {
```

Comment:
s/addExemplarToSlice/convertExemplar?

Reply:
Done, modified to `convertToOtlpExemplar`